### ISSUE TITLE
Firefox gecko driver cannot start with python3 and new selenium2library

### ISSUE THREAD

**Issue by chaussen** [robotframework/SeleniumLibrary#878]
ARCH LINUX 4.12.8-2-ARCH
geckodriver 0.18.0
firefox 55.0.3
python 3.6.2
Robot Framework 3.0.2 (Python 3.6.2 on linux)
robotframework-selenium2library (3.0.0a3)

traceback:
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/SeleniumLibrary/__init__.py", line 266, in run_keyword
    return DynamicCore.run_keyword(self, name, args, kwargs)
  File "/usr/lib/python3.6/site-packages/SeleniumLibrary/base/robotlibcore.py", line 97, in run_keyword
    return self.keywords[name](*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/SeleniumLibrary/keywords/browsermanagement.py", line 138, in open_browser
    ff_profile_dir, remote_url)
  File "/usr/lib/python3.6/site-packages/SeleniumLibrary/keywords/browsermanagement.py", line 596, in _make_browser
    browser = creation_func(remote, desired_capabilities, profile_dir)
  File "/usr/lib/python3.6/site-packages/SeleniumLibrary/keywords/browsermanagement.py", line 606, in _make_ff
    profile = webdriver.FirefoxProfile(profile_dir)
  File "/usr/lib/python3.6/site-packages/selenium/webdriver/firefox/firefox_profile.py", line 78, in __init__
    ignore=shutil.ignore_patterns("parent.lock", "lock", ".parentlock"))
  File "/usr/lib/python3.6/shutil.py", line 309, in copytree
    names = os.listdir(src)
```

cheers,
john

**Comment by aaltat** [comment]
Thanks from the report. Could you please tell how you use `Open Browser` keyword and was the Selenium version you are using?

**Comment by chaussen** [comment]
selenium (3.5.0)
Open Browser      ${URL}   browser=ff

**Comment by pekkaklarck** [comment]
1. What's the actual exception you got? The included traceback shows where the exception occurs but not what the exception is.

2. Could you test with other Python 3 versions than 3.6?

3. Could you test this with Selenium only?

**Comment by aaltat** [comment]
I can reproduce this problem with Python 3.5 Selenium2Library. Need to think what causes the problem.

**Comment by aaltat** [comment]
Fixed in #881
